README.md

```diff
@@ -447,6 +447,9 @@ update_foo(Kubeclient::Resource.new({metadata: {name: 'name', ...}, ...}))  # gl
 
 patch_foo('name', patch, 'namespace')    # namespaced
 patch_foo('name', patch)                 # global
+
+apply_foo(Kubeclient::Resource.new({metadata: {name: 'name', namespace: 'namespace', ...}, ...}), 'myapp')
+apply_foo(Kubeclient::Resource.new({metadata: {name: 'name', ...}, ...}), 'myapp')  # global
```

Comment:
I think these should say `field_manager: 'myapp'` right?

Should we mention optional `force:` here?  The focus here is to summarize how name & namespace are passed, not all options.
So let's use `, field_manager: 'myapp', **opts)`, or just `, **opts)`.